src/components/Radios.test.tsx

```diff
@@ -0,0 +1,696 @@
+import React from "react";
```

Comment:
This could probably be a lot smaller (and easier to follow) if you went with an `with all props` vs `without optional props` pattern. The extra bits to test would be places where combinations of props matter.

It would also be nice to see a test for `onChange` functioning correctly.